ory/summit-platform
# fix: add live page

Adds a `/live` page for the day of the summit. The page is locked if you aren't registered / logged in and if it's not the day of the summit. On the day of the summit, you'll see details for the currently active talk. If it's before the summit, you'll see the first talks details and after the summit you'll see the last talk's details.

In order to test this, I recommend you

1. Tunnel your playground project `ory tunnel http://localhost:3000 --project <slug>`. Make sure the schema matches the actual console schema!
2. Specify the corresponding URLs in your `.env.local`. You'll also need a few API keys. Check with me or @mszekiel to get them
3. Go to `src/app/(summit-pages)/startingDate.ts` and change the date to the current date (affects countdown and live detection)
4. Run locally and enjoy!

Closes #84
